README.md

```diff
@@ -134,14 +134,14 @@ response instead.
 1. Create a `main.py` file with the following contents:
 
    ```python
-   def hello(request):
-        return "Hello world!"
+   def hello(data, context):
```

Comment:
```suggestion
   def hello(event, context):
```

Just to be consistent with the Python example: https://cloud.google.com/functions/docs/tutorials/pubsub#functions-prepare-environment-python